.eslintignore

```diff
@@ -0,0 +1,5 @@
+node_modules/
+build/
+package-lock.json
+tsconfig.json
```

Comment:
why not formatting `tsconfig.json` as well?

Reply:
Because JSON linting strips comments. Currently there are none, but I reckon there might be.

Reply:
`tsconfig` is pretty self explanatory, let's keep it formatted.